src/renderer/components/modals/DepositWithdrawModal.vue

```diff
@@ -66,6 +69,43 @@ export default {
         this.holding.balance.toString() :
         this.holding.contractBalance.toString();
     },
+
+    cleanAmount() {
```

Comment:
Later we should move this logic into a helper method somewhere if it is shared with send code.